apps/dashboard/app/models/batch_connect/session.rb

```diff
@@ -295,7 +295,7 @@ def script(opts = {})
       content = opts.fetch(:content, "")
       options = opts.fetch(:options, {})
 
-      OodCore::Job::Script.new(options.merge(content: content))
+      OodCore::Job::Script.new(**options.merge(content: content))
```

Comment:
@johrstrom If we begin testing ood_core against Ruby 3.0, likely will need to utilize this change to support Ruby 3.0 when trying to pass hash to methods using only keyword arguments.  Seems Ruby 2.7 deprecated automatic conversion from hash to keyword arguments and it was removed in Ruby 3.0.

Reply:
Yea, we've been seeing those messages. But just using splat is an easy enough fix.